README.md

```diff
@@ -146,3 +176,4 @@ to build.
 * `mvn install`         -- Install the project into your own Maven repository.
 * `mvn site`            -- Generate Maven informational pages.
 * `mvn javadoc:javadoc` -- Generate JavaDoc.
+
```

Comment:
README is missing info on how to use custom metrics (core service) and would benefit from a link to tracing plugin docs

Reply:
I added this documentation.